lasagne/nonlinearities.py

```diff
@@ -46,6 +78,18 @@ def __call__(self, x):
 
 # linear
 def linear(x):
+    """Linear activation function :math:`\\varphi(x) = x`
+
+    Parameters
+    ----------
+    x : float32
+        The activation (the summed, weigthed input of a neuron).
+
+    Returns
+    -------
+    float32
+        The output of the rectify function applied to the activation.
```

Comment:
copy-paste error here I guess, this should read 'linear function' instead of 'rectify function'.